services/submission/src/main/resources/application.yaml

```diff
@@ -77,34 +77,27 @@ spring:
 # Actuator configuration
 management:
   server:
+    ssl:
+      enabled: false
     port: 8081
```

Comment:
@hilmarf @ioangut Do we want the actuator on a separate port or on the same as the service itself?

Reply:
@johanneseschrig - any best practices? How have you come to that port?

Reply:
port: should be 8081 which shouldn't be accessible from outside